styleguide/voice-tone.md

```diff
@@ -70,9 +73,18 @@ they may not have the English vocabulary you have.
 
 However, you are still writing for technical professionals. You can assume that your readers
 have programming knowledge and the specific vocabulary for programming terms. Object Oriented
-Programming, Class, and Object, Function and Method will all be familiar terms. However, not everyone
-reading your articles will have a formal computer science degree. Terms like 'idempotent' probably
+Programming, Class, and Object, Function and Method are all be familiar terms. However, not everyone
+reading your articles have a formal computer science degree. Terms like 'idempotent' probably
 need to be defined if you use it:
 
 >The Close() method is idempotent, meaning that you can call it multiple times and the effect is
 the same as if you called it once.
+
+### Avoid split infinitives
+While to boldly go may sound good on TV, split infinitives can reduce the readability of your content. So, to go boldly...
+
+### Avoid future tense
+In some non-English languages the concept of future tense is not the same as English. Using future tense can make your docuemnts harder to read. Additionally, when using the future tense, the obvious question is when. So if you say 'Learning PowerShell will be good for you" - the obvious question for the reader is when will it be good? Instead, just say "Learning PowerShell is good for you".
+
+### What is it - so what?
```

Comment:
This is a great point.  

@mairaw Should we highlight this more prominently?

Reply:
In terms of your questions about warnings, @doctordns, the warnings are not only from your PR, but from all the content published from the gitHub/dotnet repo; we're slowly working on addressing all of these issues. Ordinarily, the publishing process (the item that reads "OpenPublishing.Build" on the Conversions tab of the PR) will report whether your PR passed or succeeded with warnings, although it's not always accurate. It's still worthwhile to scan the actual report to see if any of the warnings might have been caused by your PR.